### ISSUE TITLE
jupyter. createnewnotebook command not found

### ISSUE THREAD

**Issue by BigZG** [microsoft/vscode-jupyter#8264]
Issue Type: <b>Bug</b>

1.Jupyter extension version : V2021.11.1..1461390
2.python environment: 
    (1)python 3.10.0
    (2)conda python 3.9.7
    (3)conda python 3.8.12
    (4)conda python 3.8.11

3. steps:
    use ctrl + shift + p to open the Command Palette,and then select Jupyter: Create New NoteBook.And after the above  operation, vscode insider show me a error message box:jupyter. createnewnotebook command not found. But at the same environment, the V2021.10.1101450599 version can work correcttly

Extension version: 2021.10.1101450599
VS Code version: Code - Insiders 1.63.0-insider (035eaab2cb31798c6ae90a35928c62117cb52fcd, 2021-11-15T05:17:52.250Z)
OS version: Windows_NT x64 10.0.22494
Restricted Mode: No

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 7 5800 8-Core Processor               (16 x 3394)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|47.87GB (38.88GB free)|
|Process Argv|--crash-reporter-id 6d524dd5-0cb3-47bc-ab6c-30ad10e659b9|
|Screen Reader|no|
|VM|0%|
</details><details>
<summary>A/B Experiments</summary>

```
vsliv695:30137379
vsins829:30139715
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythontb:30258533
pythonptprofiler:30281269
vshan820:30294714
pythondataviewer:30285072
vscod805:30301674
pythonvspyt200:30323110
vsccppwt:30382699
bridge0708:30335490
bridge0723:30353136
pythonrunftest32:30365365
pythonf5test824:30361779
javagetstartedt:30350119
pythonvspyt187:30365360
vsaa593cf:30376535
vsc1dst:30396469
vscexrecpromptt2:30397559

```

</details>

**Comment by DonJayamanne** [comment]
I'm sorry for not getting back to you.
Please could you uninstall the Jupyter extension and exit vscode completely & then re-isntall it & try this again.
Looks like the installation may have been corrupted.
If the problem persists, please could you provide the output from the `console window` (go into the command `Toggle Developer Tools` to get there)

**Comment by youguo** [comment]
我也遇到了同样的问题。

-   VS Code version: 1.63.0-insider
-   Jupyter Extension version (available under the Extensions sidebar): v2021.11.1001480274 (lastest)
-   Python Extension version (available under the Extensions sidebar): v2021.12.1482249642-dev
-   OS (Windows | Mac | Linux distro) and version: M1 Monterey 12.0.1 (21A559)
-   Python and/or Anaconda version: Python 3.8.8
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
-   Jupyter server running: Local

## Expected behaviour

vscode extension ms-toolsai.jupyter extension works well.

## Actual behaviour

command 'jupyter.selectJupyterInterpreter' not found.

ms-toolsai.jupyter loading failed.

<img width="1440" alt="iShot2021-11-20 22 41 33" src="https://user-images.githubusercontent.com/60578211/142730430-7a3edcc1-096f-484f-bf36-2f9c3f74f8d5.png">

## Steps to reproduce
Install the extension from the store and it can't work.（My jupyter notebook works well.）

**Comment by youguo** [comment]
I also encountered the same problem.

-   VS Code version: 1.63.0-insider
-   Jupyter Extension version (available under the Extensions sidebar): v2021.11.1001480274 (lastest)
-   Python Extension version (available under the Extensions sidebar): v2021.12.1482249642-dev
-   OS (Windows | Mac | Linux distro) and version: M1 Monterey 12.0.1 (21A559)
-   Python and/or Anaconda version: Python 3.8.8
-   Type of virtual environment used (N/A | venv | virtualenv | conda | ...): conda
-   Jupyter server running: Local

## Expected behaviour

vscode extension ms-toolsai.jupyter extension works well.

## Actual behaviour

command 'jupyter.selectJupyterInterpreter' not found.

ms-toolsai.jupyter loading failed.

<img width="1440" alt="iShot2021-11-20 22 41 33" src="https://user-images.githubusercontent.com/60578211/142730430-7a3edcc1-096f-484f-bf36-2f9c3f74f8d5.png">

## Steps to reproduce
Install the extension from the store and it can't work.（My jupyter notebook works well.）

**Comment by DonJayamanne** [comment]
Found the problem, will be releasing a fix shortly.
